code/datums/abilities/vampire/vampire_bite.dm

```diff
@@ -83,8 +83,8 @@
 			boutput(M, __red("The blood of the dead provides little sustenance..."))
 
 		var/bitesize = 5 * mult
-		M.change_vampire_blood(bitesize, 1)
-		M.change_vampire_blood(bitesize, 0)
+		H.change_vampire_blood(bitesize, 1)
```

Comment:
`H` for `holder` isn't very clear (at least, not to me!). Still, seems to be used quite a lot, so up to you if it's worth the effort to rename